magento/magento2
# FR#7689_22 quote_address_id not copied when converting quote address to order address

When an order is created the quote address id are not saving in the order address table.

### Description
I modified the place order function for save the quote_addres_id in order_address table properly. It was necessary to modify the Order Address Api interface for adding new getters and setters methods and Quote Management Model Logic.

### Fixed Issues (if relevant)
1. magento/magento2#7689: quote_address_id not copied when converting quote address to order address

### Manual testing scenarios
1. Add products to the cart
2. Place a guest order 
3. Check table order_address and detect that quote_addres_id is empty

With these modifications this field is saving properly.

### Related PRs
#11614

### Contribution checklist
